pkgs/development/compilers/elm/default.nix

```diff
@@ -91,11 +91,11 @@ let
           inherit nodejs pkgs;
           inherit (stdenv.hostPlatform) system;
         };
-    in with hsPkgs.elmPkgs; rec {
+    in with hsPkgs.elmPkgs; {
       elm-test = patchBinwrap [elmi-to-json] nodePkgs.elm-test;
       elm-verify-examples = patchBinwrap [elmi-to-json] nodePkgs.elm-verify-examples;
       elm-coverage =
-        let patched = patchBinwrap [elm elmi-to-json] nodePkgs.elm-coverage;
```

Comment:
this was not needed. npm installer doesn't use binwrap but it's own mechanism to download binary - this is already solved in `preBuild` hook.